libtorch-ffi-helper/libtorch-ffi-helper.cabal

```diff
@@ -1,4 +1,4 @@
-cabal-version:       2.1
+cabal-version:       3.0
```

Comment:
https://github.com/haskell/cabal/blob/6d697993e1c5b4fb036e47add0ec5d0ac9fcf474/Cabal/src/Distribution/CabalSpecVersion.hs#L67-L86

cabal-3.4 does not support cabal-2.1 and inequalities of version.